README.md

```diff
@@ -2,31 +2,32 @@
 
 Thank you for choosing us to guide you through your journey to become a software developer! This **Foundations Course** is for students who have enrolled in either a day or evening Web Development cohort.
 
-We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this couse to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
+We want to ensure that you are as prepared as possible for the technologies and concepts that your instruction staff will be coaching you through in the first few weeks, so we have prepared this course to set you up for success. These resources will provide you with the basic skills, terminology, and context for what it will be like at Nashville Software School.
 
 ## Foundations Course is Mandatory
 
-If you don't do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
+If you do not do the **Foundations Course**, you will be immediately behind your teammates on the first day of class and are at risk of needing to move to the next cohort so that you have time to complete it.
```

Comment:
I am struggling with this language. Do you know if there is a reason we say "behind your teammates". We spend so much time trying to keep them from comparing themselves to others. What if it said `If you do not complete the **Foundations Course**, you will be unprepared on the first day of class. Without this base of knowledge before you begin, you will be at risk of need to move to the following cohort pending your completion.` ?

Reply:
Hmm, I'm not sure what the reasoning was behind that particular phrasing, but I do see what you are saying. It does feel kind of 'peer-pressure-y' in a way that, as you said, sort of contradicts the safe and welcoming learning environment of NSS. Pushing up changes in just a few!